test/e2e/puppeteer.js

```diff
@@ -204,8 +207,16 @@ async function downloadLatestChromium() {
 
 	const browserFetcher = new BrowserFetcher( { path: 'test/e2e/chromium' } );
 
-	const lastRevisionURL = LAST_REVISION_URLS[ browserFetcher.platform() ];
-	const revision = await ( await fetch( lastRevisionURL ) ).text();
+	const os = PLATFORMS[ browserFetcher.platform() ];
+
+	const revisions = await ( await fetch( OMAHA_PROXY ) ).json();
+	let revision = revisions.find( revs => revs.os === os ).versions.find( version => version.channel === chromiumChannel ).branch_base_position;
+
+	while ( ! ( await browserFetcher.canDownload( revision ) ) ) {
```

Comment:
I'm afraid this is quite slow -- it must make around 50-100 HEAD requests until it will find an available snapshot...

Reply:
No, it completes in around 1 second!